pkg/controller/common/annotation/controller_version_test.go

```diff
@@ -275,6 +275,23 @@ func TestNewerAnnotation(t *testing.T) {
 	assert.True(t, compat)
 }
 
+// Test TestInvalidAnnotation check if the annotation is valid.
```

Comment:
Uh, there are other comments for the Godoc who are terrible in this file. It would be nice to fix them in the process.

```diff
diff --git a/pkg/controller/common/annotation/controller_version_test.go b/pkg/controller/common/annotation/controller_version_test.go
index 318d13c63..a98210117 100644
--- a/pkg/controller/common/annotation/controller_version_test.go
+++ b/pkg/controller/common/annotation/controller_version_test.go
@@ -108,7 +108,7 @@ func TestCheckCompatibility(t *testing.T) {
        }
 }
 
-// Test UpdateControllerVersion updates annotation if there is an older version
+// TestAnnotationUpdated tests that an annotation is updated if there is an older version
 func TestAnnotationUpdated(t *testing.T) {
        kibana := kbv1.Kibana{
                ObjectMeta: metav1.ObjectMeta{
@@ -126,7 +126,7 @@ func TestAnnotationUpdated(t *testing.T) {
        require.Equal(t, obj.GetAnnotations()[ControllerVersionAnnotation], "1.7.0")
 }
 
-// Test UpdateControllerVersion creates an annotation even if there are no current annotations
+// TestAnnotationCreated tests that an annotation is created even if there are no current annotations
 func TestAnnotationCreated(t *testing.T) {
        kibana := kbv1.Kibana{
                ObjectMeta: metav1.ObjectMeta{
@@ -220,7 +220,6 @@ func TestMissingAnnotationNewObject(t *testing.T) {
        assert.Equal(t, MinCompatibleControllerVersion, es.Annotations[ControllerVersionAnnotation])
 }
 
-//
 func TestSameAnnotation(t *testing.T) {
        es := &esv1.Elasticsearch{
                ObjectMeta: metav1.ObjectMeta{
@@ -275,7 +274,7 @@ func TestNewerAnnotation(t *testing.T) {
        assert.True(t, compat)
 }
```

Reply:
According to your suggestion, I have optimized it.